core/src/test/java/io/paradoxical/cassieq/unittests/time/TestClock.java

```diff
@@ -49,16 +77,25 @@ public void tickSeconds(Long seconds) {
         }
     }
 
+    private boolean shouldExecute(final SleepItem peek) {
+        final Instant now = now();
+
+        return truncate(peek.getSleepUntil()).isBefore(now) ||
```

Comment:
also just realized we can make this simpler by just having a `!instant.isAfter(now)`